Gemfile

```diff
@@ -105,3 +105,6 @@ gem 'kaltura-client'
 
 gem 'sentry-raven'
 gem 'strscan', '3.0.1'
+
+gem 'ed25519'
```

Comment:
Should be fine. Please also run `bundle install` so that changes are also reflected in Gemfile.lock

Reply:
Oh sure, I installed and pushed up the lock file.  Good to go?